### ISSUE TITLE
Broken Space beer for Service Borgs

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#15487]
(WEB REPORT BY: huenererschrecker REMOTE: 172.93.109.202:7777)
Revision: 5fce9b9dc013628e39f3f8f550d85b98a6f8ab1f 
The issue was that my beer was not working as a service borg. I fiddled around with it and dragged and dropped it on a counter. The slot it was in was then not working anymore, stating that I can't store something I am not holding. Other people where able to take the beer that was on the table. Admins had to reborg me to fix that.

TL;DR I somehow was able to drag my spacebeer out of my inventory, but not realy.

I exptected that I was dragging a beerbottle on the counter. 

What happened was an inaccessible slot for what I was holding.

>Steps to reproduce if possible
The only thing I realy did was drag the beer bottle on the bartable. May or may not have something to do that I played around with it first. It was empty when I put it on the counter.

**Comment by jknpj** [comment]
Fixed
